IPPDotNetDevKitCSV3/Code/Intuit.Ipp.Core/RestCalls/RestHandler.cs

```diff
@@ -308,6 +295,17 @@ public virtual System.Net.HttpWebRequest PrepareRequest(RequestParameters reques
             // Add the Request Source header value.
             httpWebRequest.UserAgent = CoreConstants.REQUESTSOURCEHEADER;
 
+            if (this.serviceContext.IppConfiguration.Logger.UseVerboseLogging)
+            {
+                this.serviceContext.IppConfiguration.Logger.CustomLogger.Log(
+                    TraceLevel.Info,
+                    "QBooks request: {Method} {RequestUri}. RealmId: {RealmId}; Body: {Content}",
```

Comment:
Let's do similar to Xero
<img width="897" alt="image" src="https://github.com/user-attachments/assets/31b3ec13-a6a8-4b20-a82a-ecc458424ab3">

```suggestion
                    "Sending request to QBooks: {RealmId} {Method} {Uri} {@Headers} {Content}",
```

Authorization header can be just excluded